src/clients/worker.rs

```diff
@@ -271,16 +271,17 @@ impl<C: golem_client::instance::Instance + Send + Sync> WorkerClient for WorkerC
         template_id: RawTemplateId,
         auth: &CloudAuthentication,
     ) -> Result<(), GolemError> {
-        let mut base_url = self.base_url.clone();
-        base_url
-            .set_scheme("wss")
+        let mut url = self.base_url.clone();
+        url.set_scheme("wss")
```

Comment:
we will have probably just https, but i think WS scheme should be set based on HTTP scheme
http -> ws
https -> wss

Reply:
Unrelated to this PR.
I'll make  a separate PR with `ws/wss`